env0/data_sshkey.go

```diff
@@ -31,23 +32,20 @@ func dataSshKey() *schema.Resource {
 
 func dataSshKeyRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	name, nameSpecified := d.GetOk("name")
-	var sshKey client.SshKey
-	var err diag.Diagnostics
+	var sshKey *client.SshKey
+	var err error
 	if nameSpecified {
 		sshKey, err = getSshKeyByName(name, meta)
-		if err != nil {
-			return err
-		}
 	} else {
 		id, idSpecified := d.GetOk("id")
 		if !idSpecified {
 			return diag.Errorf("At lease one of 'id', 'name' must be specified")
 		}
 		sshKey, err = getSshKeyById(id, meta)
-		if err != nil {
-			return err
-		}
+	}
 
+	if err != nil {
+		return diag.Errorf("could not read ssh key: %v", err)
 	}
 
 	d.SetId(sshKey.Id)
```

Comment:
`sshKey` can be `nil` now, isn't it?

Reply:
yes! good catch!
I'll need to fix that.

Reply:
And another small one (wanted output to remain consistent):
https://github.com/env0/terraform-provider-env0/pull/306/commits/476871a0dbc418e34069d89bde0652b93fcdf7ad

Reply:
@yaronya 
3cc1e9fe8c4e40f54c46a08b503e31c528ac4357